File: Brainz/Responses/ArtistResponse.cs
namespace Brainz.Responses
{
    using System;
    using Brainz.Model;
    using Newtonsoft.Json;

    public class ArtistResponse
    {
        [JsonProperty("created")]
        public DateTimeOffset Created { get; set; }

        [JsonProperty("count")]
        public long Count { get; set; }

        [JsonProperty("offset")]
        public long Offset { get; set; }

        [JsonProperty("artists")]
        public Artist[] Artists { get; set; }
    }
}

File: go/src/responses/artistresponse.go
package responses

import (
	"time"

	model "../model"
)

// ArtistResponse encapsulates the response received from an artist search.
type ArtistResponse struct {
	Created time.Time      `json:"created"`
	Count   int            `json:"count"`
	Offset  int            `json:"offset"`
	Artists []model.Artist `json:"artists"`
}
